### PR TITLE
Convert Issue Template to YAML

### PR DESCRIPTION
**Pull request by Artiume**
Old

https://github.com/jellyfin/jellyfin/issues/new?assignees=&labels=bug&template=bug_report.md&title=

New 

https://github.com/Artiume/jellyfin/issues/new?assignees=&labels=bug%2Ctriage&template=Bug+Report.yml&title=%5BIssue%5D%3A+

### CODE REVIEW

**Comment by Artiume** [comment]
if we want to validate on the Jellyfin versioning in the template, this will inherently require more maintenance. Each release, this would require an update as well. One option is to tie it into our release system to be updated to take care of the need to update it manually.

File: .github/ISSUE_TEMPLATE/issue report.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    attributes:
+      label: Jellyfin Version
+      description: What version of Jellyfin are you running?
+      options:
+        - 10.7.7
+        - 10.7.z
+        - 10.6.4
+        - Other
+    validations:
+      required: true
```
Comment (Artiume) [inline .github/ISSUE_TEMPLATE/issue report.yml:38]: Jellyfin versioning validation

**Review by 1337joe — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue report.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
+name: Issue Report
+description: File an issue report
+title: "[Issue]: "
```
[the remark this answers is not in the document]
Comment (Artiume) [reply .github/ISSUE_TEMPLATE/issue report.yml:3]: Lots of people report issues, not necessarily bugs. If it is a bug, then we can confirm it and keep the stale bot away.

[the remark this answers is not in the document]
Comment (1337joe) [reply .github/ISSUE_TEMPLATE/issue report.yml:3]: So should the [bug] label (on line 4) be changed to an [issue] label? I'm too new to the project to have a good feel for how the labels are being used.

[the remark this answers is not in the document]
Comment (Artiume) [reply .github/ISSUE_TEMPLATE/issue report.yml:3]: we'd have to make an issue label for that to work. Might be best to just remove it and leave triage only

**Review by cvium — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**